specutils/analysis/flux.py

```diff
@@ -22,7 +22,7 @@ def line_flux(spectrum, regions=None):
     spectrum : Spectrum1D
         The spectrum object over which the summed flux will be calculated.
 
-    regions: `~specutils.utils.SpectralRegion` or list of `~specutils.utils.SpectralRegion`
+    regions : `~specutils.spectra.spectral_region.SpectralRegion` or list of `~specutils.spectra.spectral_region.SpectralRegion`
```

Comment:
```suggestion
    regions : `~specutils.spectra.SpectralRegion` or list of `~specutils.spectra.SpectralRegion`
```

Reply:
I usually like to have the full path in the source code of the docs. Because of the `~` the rendered version won't show any of this anyway and for people who read the source code I think its useful to immediately see which module this comes from. However, if you prefer is in a shorter form that's also fine with me.

Reply:
See https://travis-ci.org/astropy/specutils/jobs/488019886 
The shortened version suggested by @bsipocz causes warnings in Sphinx. Thus, I will revert these commits.

Reply:
My preference is usually to keep the namespace of the API docs: https://specutils.readthedocs.io/en/latest/api/specutils.SpectralRegion.html#specutils.SpectralRegion

Substructure underneath it can change, and I think it's advised to import from the main namespace anyway and not from the individual files.

Reply:
Either way, what @bsipocz suggested fails in Sphinx. So, I reverted and force-pushed. (Which makes in hard to see in github what exactly this comment is referring to, thus this extra comment here.)

Reply:
It was failing as actually I haven't realized that the namespace is not  `spectuils.spectra.SpectralRegion` but `specutils.SpectralRegion` even though I've link the API above...

Reply:
And I just took your changes and did not check it. OK, Pushing a new version in a minute.

Reply:
Which brings me to another pet peeve that I erupt on every few months in some random issue: Is it really worth our time to have discussions to this level of detail? My PR is clearly an improvement over NOT having the links in sphinx at all; so is this discussion really the best use of our time? Should we maybe sometimes merge not 100 % perfect PRs just to get things done? I promise that I use the time that gets freed up that way to work on other astropy issues!

Perfect is the enemy of good.

Reply:
Yeah, usually nitpicking is not worth it I agree. 
On the other hand what would be worth is to finally set up docstring testing so these and other docstring issues would be picked up automatically at the time of adding it to the codebase.